### PR TITLE
Adding the templates for 2020

### PR DESCRIPTION
**Pull request by napulen**
These are the templates curated by the publication chairs of ISMIR 2020.

Please integrate them into the official repo.

### CODE REVIEW

**Comment by napulen** [comment]
Pinging people with write-access.

@blairkan @ejhumphrey @boblsturm @geoffroypeeters @ismirweb @meinardmueller @stefan-balke

**Review by ejhumphrey — APPROVED** [review]
v. quick lgtm

File: 2020/latex/ISMIR2020template.tex [hunk 1]
```diff
@@ -0,0 +1,251 @@
+% -----------------------------------------------
+% Template for ISMIR Papers
+% 2019 version, based on previous ISMIR templates
```
Comment (f0k) [inline 2020/latex/ISMIR2020template.tex:3]: `2020 version`, maybe?

Comment (napulen) [reply 2020/latex/ISMIR2020template.tex:3]: Absolutely! Changing now.

Comment (napulen) [reply 2020/latex/ISMIR2020template.tex:3]: Should be corrected now in release `2020v2` and properly linked on the website. Thanks for the heads up.

**Pull request merged**